detection_rules/devtools.py

```diff
@@ -334,7 +334,8 @@ def prune_staging_area(target_stack_version: str, dry_run: bool, exception_list:
 
 @dev_group.command('update-lock-versions')
 @click.argument('rule-ids', nargs=-1, required=False)
-def update_lock_versions(rule_ids):
+@click.option('--force', is_flag=True, help='Force update without confirmation')
+def update_lock_versions(rule_ids: Tuple[str, ...], force: bool):
```

Comment:
a nitpick: `Tuple` typehint is a bit too click-specific here, `Sequence` would be a better choice, if it is really not a list :)

Reply:
I agree that `Sequence` is more appropriate; however, we do not use `Sequence` in the code elsewhere for type hinting and we do use `Tuple[str, ...]`. For consistency, I think we should update to Sequence in a different PR where we can modify all of the relevant type hints all at once.